sources/webpki-roots-shim/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "webpki-roots"
-version = "0.20.0"
+version = "0.21.1"
```

```suggestion
version = "0.22"
```
Or I can do this when I do the traditional pre-release updates.

---

I forgot that this is the version another dep calls out, which we have to match.